lib/aws/rails/action_mailbox/engine.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-require 'action_mailbox/engine'
-
 module Aws
   module Rails
     module ActionMailbox
       # @api private
       class Engine < ::Rails::Engine
-        config.action_mailbox.ses = ActiveSupport::OrderedOptions.new
-        config.action_mailbox.ses.s3_client_options ||= {}
+        if config.respond_to?(:action_mailbox)
+          config.action_mailbox.ses = ActiveSupport::OrderedOptions.new
+          config.action_mailbox.ses.s3_client_options ||= {}
+        end
 
         initializer 'aws-sdk-rails.mount_engine' do |app|
```

Comment:
Should this also be inside the if block?

Reply:
I guess it could be. But I think there's a larger problem about where this actually belongs